plugins/tweak/tweaks/fast-trade.h

```diff
@@ -1,59 +1,44 @@
 using namespace std;
 
+static df::interface_key fast_trade_move_key(const set<df::interface_key> *input)
+{
+    return input->count(interface_key::CURSOR_DOWN_FAST)
```

Comment:
I can see what this is doing, but I have to think. That means the code is more clever than it has to be. Please use plain if statements for the sake of readability.